requirements/requirements.dev.txt

```diff
@@ -16,7 +12,7 @@ attrs==23.1.0 \
     --hash=sha256:1f28b4522cdc2fb4256ac1a020c78acf9cba2c6b461ccd2c126f3aa8e8335d04 \
     --hash=sha256:6279836d581513a26f1bf235f9acd333bc9115683f14f7e8fae46c98fc50e015
     # via
-    #   -r requirements/requirements.in
+    #   -r requirements.in
```

Comment:
Uhh is this right CC: @jimmyfagan @loganbertram

Reply:
Weird. It's almost like it ran from the requirements dir instead of root. It's just a comment, but the behavior is unexpected. 

Even if only so we don't flip/flop this comment in future commits, this may be worth replicating in our own branch with the pip-compile command being run from the proper location.

Reply:
Worth noting, the other dependabot PR has the same comment updates.